Soccar [Game-Server]/src/nl/soccar/gameserver/model/PlayerWrapper.java

```diff
@@ -29,26 +42,56 @@ public SessionWrapper getCurrentSession() {
         return GameServer.getInstance().getSessionController().getSession(session.getRoom().getName());
     }
 
+    /**
+     * Gets the id of the player.
+     * 
+     * @return int The id of the player.
+     */
     public int getPlayerId() {
         return player.getPlayerId();
     }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)